<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Nom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Ajouter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Supprimer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Search">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Prix">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Valider">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Actualiser">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualiser</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
